ocaml/xapi/xapi_pool.ml

```diff
@@ -3590,7 +3590,16 @@ let disable_repository_proxy ~__context ~self =
 let set_uefi_certificates ~__context ~self ~value =
   match !Xapi_globs.override_uefi_certs with
   | false ->
-      raise Api_errors.(Server_error (Api_errors.operation_not_allowed, [""]))
+      raise
+        Api_errors.(
+          Server_error
+            ( Api_errors.operation_not_allowed
+            , [
+                "Setting UEFI certificates is not possible when \
+                 override_uefi_certs is false"
+              ]
+            )
+        )
```

Comment:
```suggestion
      let msg =
        "Setting UEFI certificates is not possible when override_uefi_certs is \
         false"
      in
      raise Api_errors.(Server_error (operation_not_allowed, [msg]))
```

Reply:
This will need to be explained in the xenserver docs, I'm sure some customers will try to set their own certs just because they can, and support needs to be ready to tell them it's not supported